unix/x0vncserver/CMakeLists.txt

```diff
@@ -21,6 +21,13 @@ target_include_directories(x0vncserver PUBLIC ${CMAKE_SOURCE_DIR}/unix)
 target_include_directories(x0vncserver PUBLIC ${CMAKE_SOURCE_DIR}/common)
 target_link_libraries(x0vncserver tx rfb network rdr unixcommon)
 
+# check for systemd support (socket activation)
+pkg_check_modules(LIBSYSTEMD libsystemd)
```

Comment:
We try to put all checks in the top `CMakeLists.txt`, to make it easier to see TigerVNC's dependencies.

On that note, do you think you could add this as an optional dependency to `BUILDING.txt`?